### PR TITLE
Add git and repository

### PR DESCRIPTION
**Pull request by colearendt**
Add git to the connect image
Add RPackageRepository configuration by default (is there a reason we left that off? or just forgot? I know we do it for RSP)

### CODE REVIEW

**Comment by slopp** [comment]
I'm not sure I love codifying demo here... Could we at least add a nod in the documentation to how that should be changed to your RSPM server? 

Or better yet, add the docs and a comment in the gcfg file

File: connect/rstudio-connect.gcfg [hunk 1]
```diff
@@ -32,3 +32,9 @@ Provider = password
 [Python]
 Enabled = true
 Executable = /opt/python/3.6.5/bin/python
+
+[RPackageRepository "CRAN"]
+URL = https://demo.rstudiopm.com/all/__linux__/bionic/latest
```
Comment (danielfrg) [inline connect/rstudio-connect.gcfg]: Following discussion on slack and here lets comments this lines out.

**Comment by danielfrg** [comment]
Can you also commend this line https://github.com/rstudio/rstudio-docker-products/blob/master/server-pro/conf/repos.conf as part of this PR?

**Comment by colearendt** [comment]
Apologies for the delay here! Good to go?

**Comment by danielfrg** [comment]
LGTM

**Pull request merged**